docs/03-tutorials/00-security/sec-02-setup-mtls-gateway.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Set up a mutual TLS Gateway 
+---
+
+This tutorial shows how to set up a mutual TLS Gateway and configure authentication based on certificate details.
+
+## Prerequisites
+
+To follow this tutorial, use Kyma 2.6 or higher.
```

Comment:
is that the case?

Reply:
it's not the case. Kyma ma version is not related